packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFWebViewConfigurationHostApi.m

```diff
@@ -0,0 +1,86 @@
+                       error:(FlutterError *_Nullable *_Nonnull)error {
+  WKWebViewConfiguration *webViewConfiguration = [[WKWebViewConfiguration alloc] init];
+  [self.instanceManager addInstance:webViewConfiguration withIdentifier:instanceId.longValue];
+}
+
+- (void)createFromWebViewWithIdentifier:(nonnull NSNumber *)instanceId
+                      webViewIdentifier:(nonnull NSNumber *)webViewInstanceId
+                                  error:(FlutterError *_Nullable __autoreleasing *_Nonnull)error {
+  WKWebView *webView =
+      (WKWebView *)[self.instanceManager instanceForIdentifier:webViewInstanceId.longValue];
+  [self.instanceManager addInstance:webView.configuration withIdentifier:instanceId.longValue];
+}
+
+- (void)setAllowsInlineMediaPlaybackForConfigurationWithIdentifier:(nonnull NSNumber *)instanceId
+                                                         isAllowed:(nonnull NSNumber *)allow
+                                                             error:
+                                                                 (FlutterError *_Nullable *_Nonnull)
+                                                                     error {
+  [[self webViewConfigurationForIdentifier:instanceId]
+      setAllowsInlineMediaPlayback:allow.boolValue];
+}
+
+- (void)
+    setMediaTypesRequiresUserActionForConfigurationWithIdentifier:(nonnull NSNumber *)instanceId
+                                                         forTypes:
+                                                             (nonnull NSArray<
+                                                                 FWFWKAudiovisualMediaTypeEnumData
+                                                                     *> *)types
+                                                            error:
+                                                                (FlutterError *_Nullable *_Nonnull)
+                                                                    error {
+  NSAssert(types.count, @"Types must not be empty.");
+
+  WKWebViewConfiguration *configuration =
+      (WKWebViewConfiguration *)[self webViewConfigurationForIdentifier:instanceId];
+  if (@available(iOS 10.0, *)) {
+    WKAudiovisualMediaTypes typesInt = 0;
+    for (FWFWKAudiovisualMediaTypeEnumData *data in types) {
+      typesInt |= FWFWKAudiovisualMediaTypeFromEnumData(data);
+    }
+    [configuration setMediaTypesRequiringUserActionForPlayback:typesInt];
+  } else {
+    for (FWFWKAudiovisualMediaTypeEnumData *data in types) {
+      switch (data.value) {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+        case FWFWKAudiovisualMediaTypeEnumNone:
+          configuration.requiresUserActionForMediaPlayback = false;
+          break;
+        case FWFWKAudiovisualMediaTypeEnumAudio:
+        case FWFWKAudiovisualMediaTypeEnumVideo:
+        case FWFWKAudiovisualMediaTypeEnumAll:
+          configuration.requiresUserActionForMediaPlayback = true;
```

Comment:
nits:
add a break to the end of the case. 
Maybe also add an empty default case.

Reply:
Added the `break`, but I could only add an empty default case with a semicolon

```objc
default:;

//or

default:
  ;
```